ludwig/features/set_feature.py

```diff
@@ -102,68 +102,28 @@ def add_feature_data(
 
 
 class SetInputFeature(SetBaseFeature, InputFeature):
-    def __init__(self, feature):
-        super().__init__(feature)
-
-        self.vocab = []
-        self.embedding_size = 50
-        self.representation = 'dense'
-        self.embeddings_trainable = True
-        self.pretrained_embeddings = None
-        self.embeddings_on_cpu = False
-        self.dropout = False
-        self.initializer = None
-        self.regularize = True
+    encoder = 'sparse'
 
-        _ = self.overwrite_defaults(feature)
+    def __init__(self, feature, encoder_obj=None):
+        super().__init__(feature)
```

Comment:
i guess yo udon't need it anymore as you have the explicit call to the two superclasses